src/components/UserPortal/OrganizationCard/OrganizationCard.tsx

```
@@ -41,34 +39,28 @@ import type { InterfaceOrganizationCardProps } from 'types/Organization/interfac
  * @returns The organization card component.
  */
-const userId: string | null = getItem('userId');
+const userId: string | null = getItem('Talawa-admin', 'userId');
 
 function organizationCard(props: InterfaceOrganizationCardProps): JSX.Element {
   const { t } = useTranslation('translation', {
     keyPrefix: 'users',
   });
   const { t: tCommon } = useTranslation('common');
-
+  const { getItem } = useLocalStorage();
```

ESLint

[error] 'getItem' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)